addons/mass_mailing_sms/static/tests/tours/mailing_activities_split.js

```diff
@@ -0,0 +1,39 @@
+/** @odoo-module */
+
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add('mailing_activities_split', {
+    test: true,
+    url: '/web',
+    steps: [
+        {
+            content: 'Open Activity Systray',
+            trigger: '.o-mail-ActivityMenu-counter',
+        }, {
+            content: 'Open Email Activities',
+            trigger: '.o-mail-ActivityGroup :contains("Email Marketing")',
+        }, {
+            content: 'Open Email Marketing record in the kanban view',
+            trigger: '.o_kanban_record:contains("New Email!")',
+            run: () => {
+                if ($(".o_kanban_record:contains(New SMS!)").length != 0) {
+                    console.error('SMS Marketing record should not appear in this view');
+                }
+            },
+        }, {
+            content: 'Open Activity Systray',
+            trigger: '.o-mail-ActivityMenu-counter',
+        }, {
+            content: 'Open SMS Activities',
+            trigger: '.o-mail-ActivityGroup :contains("SMS Marketing")',
+        }, {
```

Comment:
Not a big deal neither, but easier to read `:contains` without whitespace before.
Whitespace in a selector by itself usually define a parent / (non-necessarily-direct) child.
Pseudo-classes like `:not` or `:contains` affect the element at hand, so no whitespace is preferred for readability.